tests/regression/fn/fn_vertical_indirection.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2021, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <cstddef>
```

Comment:
Unneeded?

Reply:
Yeah, it was for `std::size_t`, no longer needed.